airflow/sensors/base_sensor_operator.py

```diff
@@ -58,24 +59,24 @@ class BaseSensorOperator(BaseOperator, SkipMixin):
     :type mode: str
     """
     ui_color = '#e6f1f2'
```

Comment:
In `BaseOperator` you used here `# type: str`.

Reply:
Good point. I will add type here.